src/components/Text.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { or } from 'airbnb-prop-types';
-import StyleSheet from '../stylesheet';
```

Comment:
Could you limit this PR to related changes? It's already huge without those cosmetic changes haha

Reply:
Sorry I got carried away 😅 I'll remove the less relevant parts